packages/@uppy/dashboard/src/Dashboard.tsx

```diff
@@ -1297,6 +1297,60 @@ export default class Dashboard<M extends Meta, B extends Body> extends UIPlugin<
     }
   }
 
+  #getStatusBarOpts() {
+    return {
+      hideUploadButton: this.opts.hideUploadButton,
+      hideRetryButton: this.opts.hideRetryButton,
+      hidePauseResumeButton: this.opts.hidePauseResumeButton,
+      hideCancelButton: this.opts.hideCancelButton,
+      showProgressDetails: this.opts.showProgressDetails,
+      hideAfterFinish: this.opts.hideProgressAfterFinish,
+      locale: this.opts.locale,
+      doneButtonHandler: this.opts.doneButtonHandler,
+    }
```

Comment:
nit: it might be more readable to use destructuring here, not sure.
```suggestion
    const {
      hideUploadButton,
      hideRetryButton,
      hidePauseResumeButton,
      hideCancelButton,
      showProgressDetails,
      hideAfterFinish,
      locale,
      doneButtonHandler,
    } = this.opts
    return {
      hideUploadButton,
      hideRetryButton,
      hidePauseResumeButton,
      hideCancelButton,
      showProgressDetails,
      hideAfterFinish,
      locale,
      doneButtonHandler,
    }
```